### ISSUE TITLE
Edit browser set page broken

### ISSUE THREAD

**Issue by mulholio** [Fyrd/caniuse#5218]
When viewing the edit browser set page, new filters are added by themselves in this chaotic fashion:

![image](https://user-images.githubusercontent.com/20966690/71170224-9f177a00-2252-11ea-9513-79956e2a0bb9.png)

**Comment by Fyrd** [comment]
Thanks for reporting! Bug should be fixed now. If you go back to the editor as it is, hit Save and re-edit it it should appear without all the duplicate entries.

**Comment by mulholio** [comment]
Thank you!!!

**Comment by mulholio** [comment]
Unfortunately, this area of the app is still broken.

At first, after editing my configuration, I got an error telling me that the check for `IE >= 11` was not allowed. This wiped _all_ my configuration.

After this, I got very similar behaviour to before where I couldn't edit my config and clicking Save and X did nothing:

https://www.loom.com/share/6a117a26e66c409899fe8d83f565630f

Here are my console errors:

![image](https://user-images.githubusercontent.com/20966690/73459725-436eff00-436f-11ea-9119-3ef4151763f8.png)

Chrome, latest version. MacOS latest version.
